cfgov/wellbeing/templates/wellbeing/home.html

```diff
@@ -0,0 +1,786 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block title -%}
+    Find our your financial well-being
+{%- endblock %}
+
+{% block desc -%}
+    Answer ten questions to measure your own current financial well-being and see steps you can take to improve it.
+{%- endblock %}
+
+{% block css -%}
+    {{ super() }}
+    <style>
+        .m-hero {
+            background-color: #cce3f5;
+        }
+        .m-hero_image {
+            background-image: url( '/f/images/parents_hero_760x364.original.png' );
```

Comment:
Yes, it should be. Didn't have a real hero image to use here yet, so I didn't worry about the fact that it wasn't working. I think I'll have one for testing tomorrow.